.env.example

```
@@ -1,7 +1,6 @@
-ETHERSCAN_API_KEY=
-POLYGONSCAN_API_KEY=
-MUMBAI_URL=
-MAINNET_URL=
-TEST_PRIVATE_KEY=
-PRIVATE_KEY=
-INITIALIZER=
\ No newline at end of file
+MOCHA_BAIL=false
+
+RPC_URL='http://0.0.0.0:8545'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)